posting/index.go

```diff
@@ -100,6 +115,15 @@ func (txn *Txn) addIndexMutations(ctx context.Context, t *pb.DirectedEdge, p typ
 	return nil
 }
 
+// addIndexMutations adds mutation(s) for a single term, to maintain index,
+// for all tokenizers.
+func (txn *Txn) addIndexMutations(ctx context.Context, t *pb.DirectedEdge, p types.Val,
```

Comment:
To better distinguish this method from the following one, maybe it can be renamed to addIndexMutationsForAllTokenizers?